package.json

```diff
@@ -35,7 +35,7 @@
       "src/common"
     ],
     "ext": "js",
-    "exec": "run-p dev-server build-docs"
+    "exec": "run-p dev-server"
```

Comment:
`build-docs` was taking a lot of time in laptops. I figured we don't need docs for local dev. So I removed it.